eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -59,10 +59,35 @@ stages:
                       RepoId: Azure/azure-sdk-for-js
                       WorkingDirectory: $(System.DefaultWorkingDirectory)
 
+              - ${{ if ne(artifact.skipPublishPackage, 'true') }}:
+                  - job: PrePublish
```

Comment:
Let's not add a new job just for this step. Instead let's do it in the existing TagRepository job we can do it before we tag so we can catch things earlier.